app/models/merchant_user.rb

```diff
@@ -0,0 +1,4 @@
+class MerchantUser < ApplicationRecord
```

Comment:
Saw you deleted out a MerchantUser table. Do we still need this model as well?

Reply:
Probably not. I honestly just didn't want to go crazy with deleting files. I'll delete it with my next commit.